uiowa/uiowa
# Adjust person list media format

# Updates
* Changes card media format for person lists to use `media--circle`, `media--border`, and `card__media--medium`.
* Updates the media view mode for person teaser image to use "Medium - Square."

# How to test

* `blt frontend`
* `blt ds --site=sandbox.uiowa.edu`
* View effect in people block lists: http://sandbox.local.drupal.uiowa.edu/people-directory
* View effect on people page: http://sandbox.local.drupal.uiowa.edu/people
* `blt ds --site=hawkeyemarchingband.uiowa.edu` or your fav v2 site.
* `drush @hawkeyemarchingband.local uli`
* Create a page, add a featured content paragraph and a person list paragraph.
* Check each in the grid and masonry options with and without a section background color and the "hide images" option.
